soymsg/pomsg/fallback_test.go

```diff
@@ -0,0 +1,38 @@
+package pomsg
+
+import (
+	"testing"
+	"golang.org/x/text/language"
+)
+
+func TestFallback(t *testing.T) {
+	tests := []struct{
+		name string
+		localeCode string
+		expectedTags []language.Tag
+	}{
+		{
+			name: "When given a generic locale code, no extra fallbacks are provided",
+			localeCode: "en",
+			expectedTags: []language.Tag{language.English},
+		},
+		{
```

Comment:
Do you mind adding a couple more cases? For example, invalid tag, one that uses a script (chinese)

Reply:
Done

Reply:
I don't believe either of the ZZ or Zzzz branches are covered.. maybe one more case that covers them?